README.md

```diff
@@ -127,6 +128,21 @@ Notes:
   automatically track `inAppClick`  events or handle custom CTAs. This will impact
   analytics for all Safari and mobile iOS users.
 
+## `baseIterableRequest`
+
```

Comment:
Think it'd be good to have a short subtext/description here, like what we have following the other method headers